### PR TITLE
fix: kiwan alerts

### PR DESCRIPTION
**Pull request by mastudillot**
This PR resolve Kiuwan alerts. There is a false positive that requires the alert to be muted.

### Before
image

### After
image
image

### CODE REVIEW

**Review by mvarlic — DISMISSED** [review]
lgtm!

**Review by DiegoHBustamante — DISMISSED** [review]

File: Controller/Transaction/AuthorizeOneclick.php [hunk 1]
```diff
@@ -121,7 +123,7 @@ public function execute()
             $inscriptionId = intval($request['inscription']);
 
             return $this->handleOneclickRequest($inscriptionId);
-        } catch (Exception $e) {
+        } catch (InvalidRequestException | MissingArgumentException | MallTransactionAuthorizeException $e) {
```
Comment (Matiasnickolas) [inline Controller/Transaction/AuthorizeOneclick.php]: this can throw `GuzzleException`

File: Controller/Transaction/CommitWebpay.php [hunk 2]
```diff
@@ -78,7 +80,7 @@ public function execute()
             $this->log->logInfo('Request: payload -> ' . json_encode($request));
 
             return $this->handleRequest($request);
-        } catch (Exception $exception) {
+        } catch (MissingArgumentException | TransactionCommitException $exception) {
```
Comment (Matiasnickolas) [inline Controller/Transaction/CommitWebpay.php]: `TransactionCommitException` is catched in `commitTransaction` method, is unnecessary catch here.
commit method from transbank-sdk can throw `GuzzleException`

**Review by Matiasnickolas — APPROVED** [review]
LGTM 🤙

**Review by mvarlic — APPROVED** [review]
LGTM!

**Pull request merged**